src/main/java/net/minecraftforge/items/wrapper/SidedInvWrapper.java

```diff
@@ -63,7 +63,7 @@ public boolean equals(Object o)
     public int hashCode()
     {
         int result = inv.hashCode();
-        result = 31 * result + side.hashCode();
+        result = 31 * result + (side == null ? 0 : side.hashCode());
```

Comment:
You can just use the null-safe `hashCode` from `java.util.Objects` here:
```suggestion
        result = 31 * result + Objects.hashCode(side);
```